src/guide/develop/client_design.md

```diff
@@ -67,8 +67,9 @@ classDiagram
         + RetentionPolicy[] ShowRetentionPolicies(String database)
         + void DropRetentionPolicy(String database, String retentionPolicy)
         + RetentionPolicy[] ShowRetentionPolicies(String database)
-        + String[] ShowMeasurements(String database, String retentionPolicy)
+        + void CreateMeasurement(CreateMeasurementBuilder builder)
```

Comment:
There are issues with lines 67 and 69, as they are duplicated. Couly you please resolve this?